packages/swapper/src/swappers/RelaySwapper/utils/types.ts

```diff
@@ -0,0 +1,90 @@
+import type { Execute } from '@reservoir0x/relay-sdk'
+import type { ChainId } from '@shapeshiftoss/caip'
+import type { Address } from 'viem'
+
+import type { TradeQuote, TradeRate } from '../../../types'
+
+export interface RelayTradeQuote extends TradeQuote {
+  selectedRelayRoute?: Execute
+}
+export interface RelayTradeRate extends TradeRate {
+  selectedRelayRoute?: Execute
+}
```

Comment:
Not true anymore following https://github.com/shapeshift/web/pull/9188/commits/448391212005a43def0ab43b38186507be95ff6c, remove type alias as it's now irrelevant.

```suggestion
```

Reply:
Done in [a744733](https://github.com/shapeshift/web/pull/9188/commits/a7447335ccd0e1a6ea584e3bb2c988a6b7d92092)